src/ui-mapbox/index.android.ts

```diff
@@ -170,7 +171,7 @@ export class MapboxView extends MapboxViewBase {
      *
      * @see Mapbox
      */
-    public getMapboxApi(): any {
+    public getMapboxApi() {
```

Comment:
This is admittedly an unrelated change. The lack of types here have been frustrating me when using the package.